userguide/sharing.rst

```diff
@@ -1006,131 +1006,136 @@ for more details.
+   | streams_depot       | **Experimental** module to store alternate data streams in a central directory.                                                 |
+   |                     | The association with the primary file can be lost due to inode numbers changing when a directory is copied to a new location    |
+   |                     | See `<https://marc.info/?l=samba&m=132542069802160&w=2>`__.                                                                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | streams_xattr       | Enabled by default. Enable storing NTFS alternate data streams in the file system.                                              |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | syncops             | Ensure metadata operations are performed synchronously.                                                                         |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | time_audit          | Log system calls that take longer than the number of defined milliseconds.                                                      |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | unityed_media       | Allow multiple Avid clients to share a network drive.                                                                           |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | virusfilter         | This extremely **experimental** module is still under development and does not work at this time.                               |
```

Comment:
Why are we listing this if people can't use it?

Reply:
Looks like it is going to be removed for U4 - will update PR when that change is confirmed.